### DISCUSSION TITLE
IMDL may have helped to break records

### DISCUSSION THREAD

**Discussion by mr-bo-jangles** [casey/intermodal#497] [Show and tell]
Hopefully this is something you'd be interested in, recently IMDL was used to generate a torrent covering ~1.6TB of data across 500,000 files. The resulting torrent ended up being 63.5MB in size and can only be handled by a couple of the well known torrent clients (Vuze, Deluge, qBittorent and rtorrent).

**Comment by casey** [comment]
Nice!

I just checked, and there are larger torrents. On one tracker I'm on, the largest torrent is 1.84 TB. I wonder what the largest torrent of all time is?

**Reply by mr-bo-jangles** [reply]
Well we discovered we were missing a lot of files from our archive, so we'll be back at some point with a larger one (then we'd just need a torrent client that can handle it 😉)
